.npmignore

```diff
@@ -0,0 +1,30 @@
+# Logs
```

Comment:
I might need to update the slush generator but it's better if our `.npmignore`s ignore everything then unignore anything necessary. Like https://github.com/Teamwork/chat-renderer/blob/master/.npmignore